deployment/terraform/assets/cluster.tf

```diff
@@ -496,13 +497,7 @@ resource "aws_security_group" "elastic" {
     security_groups = ["${aws_security_group.app.id}", "${aws_security_group.metrics.id}"]
   }
 
-  egress {
```

Comment:
I've been re-checking the need for this, taking into account the feedback received in the last PR, and is not need.